src/runtime_src/tools/scripts/pkg_hw_platform.sh

```diff
@@ -503,12 +504,19 @@ initXsaBinEnvAndVars()
           if [ "${1}" != "NOT_DEFINED" ]; then
             SchedulerFamily="${1}"
           fi
+          
+          # Sys Mod
+          set -- `cat "${metaDataJSONFile}" | python -c "import sys, json; print json.load(sys.stdin)['xsabin'].get('Sys Mode','NOT_DEFINED')"`
```

Comment:
Why is this additional information being stored in the ext_metadata file?

This file is meant to be use only to help package additional data.  Not to drive xclbinutil option values.